src/clog/TraceEmitterModules/CLogSTDOUT.cs

```diff
@@ -249,10 +122,10 @@ public void TraceLineDiscovered(string sourceFile, CLogOutputInfo outputInfo, CL
                         printf += "%u";
                         break;
                     case CLogEncodingType.Int64:
-                        printf += "%lld";
+                        printf += "%ld";
```

Comment:
Why did you change these? `ll` == "long long" == 64-bit

Reply:
agreed; fixed